SwiftSH/Command.swift

```diff
@@ -92,54 +96,56 @@ public class SSHCommand<T: RawLibrary>: SSHChannel<T> {
                 throw SSHError.allocation
             }
 
-            socketSource.setEventHandler { [unowned self] in
+            socketSource.setEventHandler { [weak self] in
                 // Suspend the timer to prevent calling completion two times
-                self.timeoutSource!.suspend()
+                self?.timeoutSource!.suspend()
```

Comment:
Instead of having an optional everywhere, can you do something like this?
```Swift
socketSource.setEventHandler { [weak self] in
    guard let command = self else {
        return
    }
    
    // Suspend the timer to prevent calling completion two times
    command.timeoutSource!.suspend()

    // [...]
}
```

Reply:
Sure, or I can call it strongSelf, that is quite common as well. Take your pick!

Reply:
I don't mind the name! I usually use `strongSelf`, so I'm happy with that 😄